plantplaces/src/main/java/com/finansir/dto/FinanceDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.finansir.dto;
+
+public class FinanceDTO {
+	
+	private int amount;
+	private int amountDue;
+	private String dueDate;
+	public int getAmount() {
+		return amount;
+	}
+	public void setAmount(int amount) {
+		this.amount = amount;
+	}
+	public int getAmountDue() {
+		return amountDue;
+	}
+	public void setAmountDue(int amountDue) {
+		this.amountDue = amountDue;
+	}
+	public String getDueDate() {
+		return dueDate;
+	}
+	public void setDueDate(String dueDate) {
+		this.dueDate = dueDate;

```

Comment:
Added a DTO class to handle data, added variables with the needed getters and setters. 
you can change the variables to whatever fits your project and naming conventions.